### ISSUE TITLE
Incorrect y position of underscore cursor?

### ISSUE THREAD

**Issue by kangjianbin** [mintty/mintty#589]
Hi mintty experts,

The y position of underscore cursor looks incorrect.
The cursor in mintty (looks like a strikeout in '~~g~~'):
<img width="151" alt="mintty" src="https://cloud.githubusercontent.com/assets/121803/18536272/6876bdd6-7b2e-11e6-9816-efdbf64eed55.png">

The cursor in emacs:
<img width="158" alt="emacs" src="https://cloud.githubusercontent.com/assets/121803/18536276/6c7eea16-7b2e-11e6-8c2f-a1de495d7b78.png">

Seems that emacs' style is better.

**Comment by mintty** [comment]
Underscore position depends on a number of factors; font, font size, row spacing, and whether underscore is drawn by mintty or Windows (which again depends...). I will check the algorithm to place the self-drawn underscore, but to assess your report, please provide your full configuration.

**Comment by kangjianbin** [comment]
My configuration is:
```
ForegroundColour=101,123,131
BackgroundColour=253,246,227
CursorColour=205,150,205
Black=7,54,66
BoldBlack=0,43,54
Red=220,50,47
BoldRed=203,75,22
Green=133,153,0
BoldGreen=88,110,117
Yellow=181,137,0
BoldYellow=101,123,131
Blue=38,139,210
BoldBlue=131,148,150
Magenta=211,54,130
BoldMagenta=108,113,196
Cyan=42,161,152
BoldCyan=147,161,161
White=238,232,213
BoldWhite=253,246,227
BoldAsFont=yes
CursorType=block
CursorBlinks=no
Font=Monaco
FontHeight=12
BoldAsColour=no
FontSmoothing=default
RowSpacing=3
AllowSetSelection=yes
ClicksPlaceCursor=no
Transparency=off
OpaqueWhenFocused=no
BackspaceSendsBS=no
Scrollbar=none
```

**Comment by mintty** [comment]
[stripped down config example a little bit]
The previous comment actually referred to the underscore text attribute, not the cursor.
For both, however, I do not see the underscore as close to the baseline as in your screenshot.
With RowSpacing=3 there may be additional room to place it a little bit lower, perhaps.

**Comment by kangjianbin** [comment]
RowSpacing affects the y position. This is an screenshot with RowSpacing=20:
<img width="78" alt="1" src="https://cloud.githubusercontent.com/assets/121803/18577824/e9bbcae4-7c1e-11e6-8e7d-97a4727a9b47.png">

**Comment by mintty** [comment]
Released in 2.6.1.
